linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands/oval/shared.xml

```diff
@@ -23,23 +23,19 @@
 
   <!-- First define OVAL entities that can be reused across tests below -->
   <unix:file_object id="object_system_privileged_commands" comment="system files with setuid or setgid permission set" version="1">
-    <unix:behaviors recurse="directories" recurse_direction="down" recurse_file_system="local" max_depth="-1" />
-    <unix:path operation="equals">/</unix:path>
+    <unix:behaviors recurse_file_system="local"/>
+    <!-- don't search /dev, /proc, /sys -->
+    <unix:path operation="pattern match">^\/(?!(dev|proc|sys)(\/|$)).*$</unix:path>
     <!-- [a-z]+ regex below is a workaround for OpenSCAP https://fedorahosted.org/openscap/ticket/457 bug -->
```

Comment:
Do we have a clue what https://fedorahosted.org/openscap/ticket/457 was about?
Can we do `filename` like this?

https://github.com/ComplianceAsCode/content/blob/da57b9e50cfd6630116f169355aa14a4f5f54adf/linux_os/guide/system/logging/rsyslog_sending_messages/rsyslog_remote_loghost/oval/shared.xml#L42

Reply:
I can't find it but I have found this comment referencing it: https://bugzilla.redhat.com/show_bug.cgi?id=1237197#c2